benchflow-experiment-manager/application/src/main/java/cloud/benchflow/experimentmanager/tasks/experiment/ExperimentRunningTask.java

```diff
@@ -0,0 +1,185 @@
+package cloud.benchflow.experimentmanager.tasks.experiment;
+
+import cloud.benchflow.dsl.BenchFlowDSL;
+import cloud.benchflow.dsl.definition.BenchFlowExperiment;
+import cloud.benchflow.experimentmanager.constants.BenchFlowConstants;
+import cloud.benchflow.experimentmanager.demo.DriversMakerCompatibleID;
+import cloud.benchflow.experimentmanager.exceptions.BenchFlowExperimentIDDoesNotExistException;
+import cloud.benchflow.experimentmanager.exceptions.TrialIDDoesNotExistException;
+import cloud.benchflow.experimentmanager.models.BenchFlowExperimentModel;
+import cloud.benchflow.experimentmanager.models.BenchFlowExperimentModel.BenchFlowExperimentState;
+import cloud.benchflow.experimentmanager.models.BenchFlowExperimentModel.BenchFlowExperimentStatus;
+import cloud.benchflow.experimentmanager.services.external.BenchFlowTestManagerService;
+import cloud.benchflow.experimentmanager.services.external.MinioService;
+import cloud.benchflow.experimentmanager.services.internal.dao.BenchFlowExperimentModelDAO;
+import cloud.benchflow.experimentmanager.tasks.CancellableTask;
+import cloud.benchflow.faban.client.FabanClient;
+import cloud.benchflow.faban.client.exceptions.ConfigFileNotFoundException;
+import cloud.benchflow.faban.client.exceptions.FabanClientException;
+import cloud.benchflow.faban.client.exceptions.RunIdNotFoundException;
+import cloud.benchflow.faban.client.responses.RunId;
+import cloud.benchflow.faban.client.responses.RunStatus;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Paths;
+
+import static cloud.benchflow.experimentmanager.constants.BenchFlowConstants.MODEL_ID_DELIMITER;
+import static cloud.benchflow.faban.client.responses.RunStatus.Code.*;
+
+/**
+ * @author Jesper Findahl (jesper.findahl@usi.ch)
+ *         created on 2017-04-19
+ */
+public class ExperimentRunningTask extends CancellableTask {
+
+    private static final String TEMP_DIR = "./tmp";
```

Comment:
If you look at the usage (https://github.com/benchflow/benchflow/pull/241/files/20a81db7f54fbd40836063bbd0b09402dd338bae#diff-343ab7ec46b9117a2b4e7ed28229f9a6R101) you can see that in addition to this there is a unique path based on the experimentID, so I think it should be fine.